### PR TITLE
HTBHF-2174 add select address acceptance tests

### PR DESCRIPTION
**Pull request by gethyn1**
Happy 😊 and sad 😞 path user journey acceptance tests for postcode lookup on select address page.

### CODE REVIEW

File: src/test/acceptance/features/address-lookup.feature [hunk 1]
```diff
@@ -3,22 +3,36 @@ Feature: Select address
...
 
-  Background:
+  # TODO DW HTBHF-2037 include test once address lookup is enabled
+  @ignore
+  Scenario: Entering a postcode with no search results shows that no addresses were found
     Given I have entered my details up to the postcode page
+    When I enter a postcode with no search results
+    Then I am shown the select address page
+    And I am informed that no addresses were found for my postcode
+    And I am shown a button to enter my address manually
```
Comment (YetAnotherMatt) [inline src/test/acceptance/features/address-lookup.feature]: And I am shown a link to change my postcode?

File: src/test/acceptance/features/address-lookup.feature [hunk 2]
```diff
@@ -3,22 +3,36 @@ Feature: Select address
   As a potential claimant
   I want to lookup my address
 
-  Background:
+  # TODO DW HTBHF-2037 include test once address lookup is enabled
+  @ignore
+  Scenario: Entering a postcode with no search results shows that no addresses were found
     Given I have entered my details up to the postcode page
+    When I enter a postcode with no search results
```
Comment (mikeSmith08) [inline src/test/acceptance/features/address-lookup.feature]: Maybe `When I enter a postcode that returns no search results`?

File: src/test/acceptance/features/address-lookup.feature (hunk 1 above)
Comment (gethyn1) [reply src/test/acceptance/features/address-lookup.feature]: Added to the feature

File: src/test/acceptance/features/address-lookup.feature (hunk 2 above)
Comment (gethyn1) [reply src/test/acceptance/features/address-lookup.feature]: Updated

File: src/test/acceptance/features/address-lookup.feature [hunk 3]
```diff
@@ -3,22 +3,37 @@ Feature: Select address
...
-    # TODO DW HTBHF-2037 include test once address lookup is enabled
-    @ignore
-    Scenario: Entering a postcode with no search results shows that no addresses were found
-      When I enter a postcode with no search results
-      Then I am shown the select address page
-      And I am informed that no addresses were found for my postcode
-      And I am shown a button to enter my address manually
+  # TODO DW HTBHF-2037 include test once address lookup is enabled
+  @ignore
+  Scenario: Entering a postcode shows a list of matching addresses
```
Comment (mikeSmith08) [inline src/test/acceptance/features/address-lookup.feature]: I do think postcode and select-address tests should be in their own feature file as we'll definitely need it when we come to the postcode validation story.

**Pull request merged**